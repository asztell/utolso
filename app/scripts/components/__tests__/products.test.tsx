import React from "react";
import { render } from "@testing-library/react";

import { Products } from "../products";

describe("<Products />", () => {
  test("render Products", () => {
    const { container } = render(<Products />);
    expect(container).toMatchSnapshot();
  });
});
