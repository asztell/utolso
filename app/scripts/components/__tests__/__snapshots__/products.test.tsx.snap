// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Products /> render Products 1`] = `<div />`;
